angular/projects/spark-angular/src/public-api.ts

```diff
@@ -118,8 +118,10 @@ export { SprkTabbedNavigationTabDirective } from './lib/directives/tabbed-naviga
 export { SprkTabbedNavigationTabModule } from './lib/directives/tabbed-navigation/sprk-tabbed-navigation-tab/sprk-tabbed-navigation-tab.module';
 export { SprkIconInputContainerComponent as ɵbu } from './lib/components/inputs/sprk-icon-input-container/sprk-icon-input-container.component';
 export { SprkIconInputContainerModule as ɵbt } from './lib/components/inputs/sprk-icon-input-container/sprk-icon-input-container.module';
-export { SparkInputContainerComponent as ɵbn } from './lib/components/inputs/sprk-input-container/sprk-input-container.component';
-export { SparkInputContainerModule as ɵbm } from './lib/components/inputs/sprk-input-container/sprk-input-container.module';
+export { SprkInputContainerComponent as ɵbn } from './lib/components/inputs/sprk-input-container/sprk-input-container.component';
+export { SprkInputContainerModule as ɵbm } from './lib/components/inputs/sprk-input-container/sprk-input-container.module';
+export { SprkInputContainerComponent as SparkInputContainerComponent } from './lib/components/inputs/sprk-input-container/sprk-input-container.component';
```

Comment:
Oh we only had the obscure named exports in here and not the originals?

Reply:
They were up top on lines 11-12. I updated those to the new name. Then added the exports for the new name as the old name.